tests/test_nc_to_mmd.py

```diff
@@ -1812,6 +1812,36 @@ def test_check_attributes_not_empty__accepts_0(self):
         ncin.subswath = 0
         self.assertEqual(None, md.check_attributes_not_empty(ncin))
 
+    def test_check_feature_type__missing(self):
+        """ Test that the correct warning is issued if the CF
+        attribute featureType is missing.
+        """
+        md = Nc_to_mmd(self.fail_nc, check_only=True)
+        ncin = Dataset(md.netcdf_product, "w", diskless=True)
+        ncin.tull = "tull"
+        md.check_feature_type(ncin)
+        self.assertEqual(
+            md.missing_attributes["warnings"][0],
```

Comment:
Testene burde KANSKJE byttes til STRING in md.missing_attributes["warnings"], for mindre hardkodet indeksering

Reply:
hva mener du?

Reply:
md.missing_attributes["warnings"][0] er en direkte indeksering, mens istedenfor kan du sjekke om en streng er et element i lista md.missing_attributes. Men dette er mer en kommentar for testinga generelt